pkg/monitoring/rules/rules-tests.yaml

```diff
@@ -170,8 +170,18 @@ tests:
   # VMStorageClassWarning alert tests
   - interval: "1m"
     input_series:
-      - series: 'kubevirt_ssp_vm_rbd_block_volume_without_rxbounce'
-        values: '0 0 1 0'
+      - series: 'kubevirt_vmi_info{name="vm1", namespace="ns1", guest_os_name="Microsoft Windows"}'
+        values: '1 1 1 0'
+      - series: 'kubevirt_vmi_info{name="vm2", namespace="ns1", guest_os_name="Red Hat Enterprise Linux"}'
```

Comment:
can you please add entries for the two labels, to verify the `or` in the expression works as expected. 
- one entry with both `guest_os_name` and `os` populated with windows values.
- one entry with both `guest_os_name` and `os` populated with non-windows values.
- one entry with `guest_os_name` populated with windows value and `os` not, and another with the opposite.